### ISSUE TITLE
Adding sound?

### ISSUE THREAD

**Issue by Raikiri** [cloudyluna/cdda-custom-experimental#2]
How do I add a soundpack to this?

**Comment by cloudyluna** [comment]
Hi @Raikiri. Thanks for opening this issue :smiley:.

If you're looking to add sound packs, say dynamically through `nix build` as a string of arguments for example, then I'm afraid I have to answer that with: I don't know :confused:. 
Maybe you could, but my experience with nix and flakes are still very limited to answer that properly :see_no_evil:.

Although, I released a new update to make adding sound packs through the nix flake code, a bit easier, as demonstrated [here](https://github.com/cloudyluna/cdda-custom-experimental/blob/8bb1f6f90bda17add681ac628a5c17c88ed24749/flake.nix#L102). Unfortunately, you'd have to clone this flake repository and locally update the `cddaExtraContents.soundPacks` to your liking, and rebuild it yourself (just a simple `nix build --refresh .#extras` would do).

**Comment by Raikiri** [comment]
I guess it's worth mentioning in the readme that you can simply copy a soundpack manually (non-declaratively) into `~/.cdda-custom-experimental` for those of us less than literate with nix.

I also think that adding some basic soundpack by default is probably better than having no soundpack at all.

**Comment by cloudyluna** [comment]
I've addressed on the adding a soundpack into the `$USERDIR/sound` in the README.md with the latest 0.3.0 release. That's cool to know. Didn't know I could do something like that.

For the default soundpack, the current game used does include a default soundpack (CC-sounds), though you'd need to enable this manually in your game options. tbf, I'm not sure why this is not yet the default. I also addressed this in the new README.md with a potential downside.
